### PR TITLE
Add support for Kafka management scripts

### PR DESCRIPTION
**Pull request by lewijacn**
### Description
This adds Kafka distribution management scripts to the Migration Console, to allow for more advanced Kafka management with this solution. As would be the case if a user wanted to move a consumer group offset back in time or further in time, delete a topic entirely, or a plethora of other abilities.
 
Pending things I am checking:
~~Any issue running this with docker solution~~, Restructured this a bit to be docker first and then AWS MSK applicable second
~~Can the IAM permissions be narrowed a bit for MSK~~, Narrowed down now
~~Is there a better way to get these Kafka distribution scripts without the 100MB file download~~, The times have been pretty quick from my experience but open to better suggestions here

### Issues Resolved
https://opensearch.atlassian.net/browse/MIGRATIONS-1332

### Testing
Manual AWS deployments

### Check List
- [ ] New functionality includes testing
  - [ ] All tests pass, including unit test, integration test and doctest
- [ ] New functionality has been documented
- [ ] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by gregschohn — APPROVED** [review]
Approved, though we'll have future work to do, either within this PR or other ones.

File: TrafficCapture/dockerSolution/src/main/docker/migrationConsole/Dockerfile [hunk 1]
```diff
@@ -3,15 +3,26 @@ FROM ubuntu:jammy
...
+COPY msk-iam-auth.properties /root/kafka-tools/aws
+COPY kafkaCmdRef.md /root/kafka-tools
 RUN chmod ug+x /root/runTestBenchmarks.sh
 RUN chmod ug+x /root/humanReadableLogs.py
 RUN chmod ug+x /root/catIndices.sh
+WORKDIR /root/kafka-tools
+RUN curl https://archive.apache.org/dist/kafka/3.5.0/kafka_2.13-3.5.0.tgz -o kafka-3-5.tgz
+# Unpackage kafka distribution and name top level 'kafka'
+RUN tar xzf kafka-3-5.tgz --transform 's!^[^/]*!kafka!'
+RUN rm kafka-3-5.tgz
```
[the remark this answers is not in the document]
Comment (lewijacn) [reply TrafficCapture/dockerSolution/src/main/docker/migrationConsole/Dockerfile]: Good callout have combined now

File: TrafficCapture/dockerSolution/src/main/docker/migrationConsole/Dockerfile [hunk 2]
```diff
@@ -3,15 +3,26 @@ FROM ubuntu:jammy
...
+RUN wget -O aws/msk-iam-auth.jar https://github.com/aws/aws-msk-iam-auth/releases/download/v1.1.9/aws-msk-iam-auth-1.1.9-all.jar
+# This is a bit opinionated and should be made cloud agnostic when this Dockerfile becomes more dynamic
+ENV CLASSPATH=/root/kafka-tools/aws/msk-iam-auth.jar
```
[the remark this answers is not in the document]
Comment (lewijacn) [reply TrafficCapture/dockerSolution/src/main/docker/migrationConsole/Dockerfile]: The scripts are in the kafka tarball. We can do this a bit cleaner by including this jar in the kafka/libs directory which I've adjusted for now 👍

File: deployment/copilot/migration-console/manifest.yml [hunk 3]
```diff
@@ -26,13 +26,14 @@ storage:
         id: ${MIGRATION_REPLAYER_OUTPUT_EFS_ID}
 
 cpu: 512       # Number of CPU units for the task.
-memory: 1024    # Amount of memory in MiB used by the task.
+memory: 2048    # Amount of memory in MiB used by the task.
```
[the remark this answers is not in the document]
Comment (lewijacn) [reply deployment/copilot/migration-console/manifest.yml]: This was a testing thing I was checking as I was getting a false OOM error with some kafka commands (believe it turned out to actually be a timeout issue). Have reverted now

**Pull request merged**